waspc/examples/todoApp/src/server/auth/google.js

```diff
@@ -7,7 +7,7 @@ export function config() {
   }
 }
 
-export async function getUserFields(_context, args) {
+export function getUserFields({ profile }) {
```

Comment:
How is it that context got removed?

Reply:
I wanted to unify the two APIs and decided to remove the `context` argument since I didn't find many uses for it personally. We can return it if people complain?